tests/data_samples/neale2_saige_study_manifest.samples.tsv

```diff
@@ -0,0 +1,11 @@
+in_path	code	trait	n_total	n_cases	trait_category
+gs://genetics-portal-raw/uk_biobank_sumstats/harmonised_split_190318/PheCode_244_SAIGE_Nov2017_MACge20.GRCh38.tsv.split*.gz	SAIGE_244	Hypothyroidism	406300	14871	endocrine/metabolic

```

Comment:
Yep, also `CommandException: One or more URLs matched no objects.` for these ones.

@d0choa @DSuveges Do you know if this might be due to the fact that I just don't have access? Or is it likely that they moved them somewhere?

(I _think_ sumstats are supposed to be public anyway, right?)